app/dashboard/templates/bounty/fulfill.html

```diff
@@ -93,6 +93,12 @@ <h4 class="text-center mb-4">{% trans "Submit Work" %}</h4>
                         {% endfor %}
                       </select>
                     </div>
+
+                  {% elif bounty.event %}
+                    <div class="mt-2">
+                      <label class="form__label" for="projectId">{% trans "Please create your project:" %}</label>
```

Comment:
@octavioamu ux review: instead of row for creating a project with "Add Project" button, may be better to include a text like "You can create a project here if you haven't" under the project URL input as a helptext with hyperlinked "create a project" to trigger the modal.

Reply:
![image](https://user-images.githubusercontent.com/1358093/96479584-8a5b9800-120f-11eb-8745-e1e187b56802.png)
check the image, the project link (in some point need to be named diff)  is just the link or where the code is.
the project select is for hackathons, contributors create a project based on their submission but is not related to the project link (but probably will have the same link)

Reply:
![image](https://user-images.githubusercontent.com/1358093/96479871-e1616d00-120f-11eb-8e8a-9a930e4eba45.png)
this is the project form

Reply:
@octavioamu okay this makes sense... 💯